### ISSUE TITLE
Viewed post history

### ISSUE THREAD

**Issue by kemmel-dev** [laurencedawson/reddit-sync-development#2150]
Reddit Sync already remembers which posts I've already opened by highlighting them in a greyed out colour. 

I was wondering if a feature could be added which allows us to view a history of posts we've opened. 

Sometimes I open Sync and view some cached posts, then I refresh the page before deciding I want to share one of the cached posts. Since I've then already reloaded the page, those posts are hard to find, as my screen is filled with new posts. It's especially hard to find these posts if you were browsing /r/all or the frontpage and don't remember which sub the post was from.

*TLDR* please add a history of viewed posts (doesn't have to be extensive, possible to cap it at like 50 max or something).
